std/meta.d

```diff
@@ -1620,3 +1621,36 @@ private template Pack(T...)
 // TODO: Consider publicly exposing this, maybe even if only for better
 // understandability of error messages.
 alias Instantiate(alias Template, Params...) = Template!Params;
+
+
+/**
+Selects a subset of the argument list by stepping with fixed stepSize over the list.
+Params:
+    stepSize = Number of elements to increment on each iteration. Needs to be greater than `0`.
+    offset = Initial offset
```

Comment:
Agreed. Using `Args[offset..$]` is not significantly more complicated, and the use of two consecutive `size_ts` practically guarantees that devs and reviewers will need to look up the argument order on every use of the function.